mlflow/pyfunc/model.py

```diff
@@ -294,6 +295,8 @@ def _save_model_with_class_artifacts_params(
     custom_model_config_kwargs = {
         CONFIG_KEY_CLOUDPICKLE_VERSION: cloudpickle.__version__,
     }
+    if model_code_path:
+        custom_model_config_kwargs[mlflow.pyfunc._MODEL_CODE_PATH] = model_code_path
```

Comment:
Can we add this as a top level arg rather than adding it in `custom_model_config_kwargs`

Reply:
hm the reason i did it this way is i'm not sure of a way to use the `MODEL_CODE_PATH` variable as the param name into `add_to_model` and didn't want another definition of `MODEL_CODE_PATH`'s value ("model_code_path")